doc/source/timeseries.rst

```diff
@@ -1473,6 +1473,28 @@ Furthermore, you can also specify multiple aggregation functions for each column
    r.agg({'A' : ['sum','std'], 'B' : ['mean','std'] })
 
 
+If a ``DataFrame`` does not have a ``DatetimeIndex``, but instead you want
+to resample based on column in the frame, it can passed to the ``on`` keyword.
```

Comment:
make it clear that the ``on`` (currently) still must be a datetimelike (so we of course accept ``PeriodIndex/TimedeltaIndex`` here as well (add tests if we don't have them for those as well)